tests/zeroex/zeroex_api_fills_test.sql

```diff
@@ -0,0 +1,12 @@
+WITH unit_tests as
+(SELECT case when test.taker_symbol = actual.taker_symbol 
+               
+                and test.maker_symbol = actual.maker_symbol
+then True else False end as test
+FROM {{ ref('zeroex_api_fills') }} actual 
+JOIN {{ ref('zeroex_api_fills') }} test 
```

Comment:
Also it seems these two ref sources are the same, and the test will always be true.

Reply:
true, but the base models each have their own tests, which is where we try to emphasize. those have been in prod for a bit.